### PR TITLE
Add Ben Tan's PPP and update docs

### PR DESCRIPTION
**Pull request by bentanjunrong**
Let's:
- Add my PPP
- Update ReadMe image and link to project docs
- Update AboutUs project member names and PPP page links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ NUS ModBook
 ===========
 [![CI Status](https://github.com/AY2122S1-CS2103T-T13-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-T13-1/tp/actions)
 
-![Ui](docs/images/Ui.png)
+![Quick Start](docs/images/QuickStartView.png)
```
Comment (rohit0718) [inline README.md:5]: any reason for changing this?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@ NUS ModBook
...
 
 NUS ModBook is a **desktop app for managing modules, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ModBook can help you organise modules faster than traditional GUI apps.
+
+Check out our [project website](https://ay2122s1-cs2103t-t13-1.github.io/tp/) to find out more!
```
Comment (rohit0718) [inline README.md:9]: dont think this is needed? this is shown as the landing page of the website so may seem weird

File: docs/AboutUs.md [hunk 3]
```diff
@@ -7,43 +7,48 @@ We are a team based in the [School of Computing, National University of Singapor
...
 
-[[GitHub](https://github.com/rohit0718)]
+Role: Documentation
 
-* Role: Documentation
+* [GitHub](https://github.com/rohit0718)
+* [Project Portfolio Page](team/rohit0718.html)
 
-### Benjamin Tan
+### TAN JUN RONG, BENJAMIN
```
Comment (rohit0718) [inline docs/AboutUs.md:37]: image

File: docs/team/bentanjunrong.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        * Added `EditModDescriptor`, `EditExamDescriptor` and `EditLessonDescriptor` classes to store the details of each edit command
+        * Added tests for all relevant classes and methods added
+        * Added utility methods to recreate command strings based on the `EditModDescriptor`, `EditExamDescriptor` and `EditLessonDescriptor`
+
+* **Enhancements to existing features**:
+    * Added flexible constructors and readable string for the Day enum [#131](https://github.com/AY2122S1-CS2103T-T13-1/tp/pull/131)
+  * Adapt `Model` interface and `ModelManager` class [#60](https://github.com/AY2122S1-CS2103T-T13-1/tp/pull/60)
+      * Added relevant CRUD methods for modules in the `ModelManager` and `ModBook` classes
+      * Implemented `UniqueModuleList` class to handle list operations for modules
+
```
Comment (rohit0718) [inline docs/team/bentanjunrong.md:31]: can standardise the bullet indentations here

**Review by rohit0718 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bentanjunrong) [reply README.md:5]: The previous UI is our sad mockup

File: docs/AboutUs.md (hunk 3 above)
Comment (bentanjunrong) [reply docs/AboutUs.md:37]: You are very rude. Please reflect on your behaviour!

File: README.md (hunk 2 above)
Comment (bentanjunrong) [reply README.md:9]: I think we should link to our docs somehow from this page?

File: docs/team/bentanjunrong.md (hunk 4 above)
Comment (bentanjunrong) [reply docs/team/bentanjunrong.md:31]: i can, but i don't want to

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -532,15 +531,39 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: First module is deleted from the list. Details of the deleted module are shown in the status message.
 
    3. Test case: `delete mod 0` <br>
-      Expected: No module is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No module is deleted. Error details shown in the status message, with correct command format provided.
+
+   4. Other incorrect `delete` commands to try:
+      1. `delete`, `delete 1` <br>
+         Expected: Similar to previous.
+      2. `delete mod x` (where x is not within the list size) <br>
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:548]: shd be x greater than list size

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -532,15 +531,39 @@ testers are expected to do more *exploratory* testing.
...
+
+   3. Test case: `edit mod 1` <br>
+      Expected: No module is edited. Error details shown in the status message, with correct command format provided.
 
-   4. Other incorrect delete commands to try: `delete`, `delete 1`, `delete mod x` (where x is larger than the list size) <br>
-      Expected: Similar to previous.
+   4. Other incorrect `edit` commands to try:
+      1. `edit`, `edit 1` <br>
+         Expected: Similar to previous.
+      2. `edit mod x c/CS1010X n/Programming Methodology` (where x is not within the list size) <br>
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:568]: same for this

**Review by rohit0718 — APPROVED** [review]
lgtm. beautiful sir. its time to finally sleep 😍

**Pull request merged**